js/src/wp-seo-term-scraper.js

```diff
@@ -19,12 +19,13 @@ var UsedKeywords = require( "./analysis/usedKeywords" );
 
 import TermDataCollector from "./analysis/TermDataCollector";
 import { termsTmceId as tmceId } from "./wp-seo-tinymce";
+import initializeEdit from "./edit";
+import { setActiveKeyword } from "./redux/actions/activeKeyword";
+import { setReadabilityResults, setSeoResultsForKeyword } from "yoast-components/composites/Plugin/ContentAnalysis/actions/contentAnalysis";
 
 window.yoastHideMarkers = true;
 
-( function( $ ) {
-	"use strict";
-
+( function( $, window ) {
```

Comment:
is passing window really necessary? shouldn't be assumed it's global?

Reply:
Please reply to this

Reply:
I don't know why but it didn't work when I called window directly without passing it directly to the function exection.